src/main/java/wolox/training/models/User.java

```diff
@@ -76,23 +81,23 @@ public LocalDate getBirthdate() {
     }
 
     public void setBirthdate(LocalDate birthdate) {
-        this.birthdate = birthdate;
+        this.birthdate = checkNotNull(birthdate);
```

Comment:
After checking if it is null, you can check if the date is after the current one (the date of birth must always be earlier).